jp2_online/templates/captura/egreso_form.html

```diff
@@ -0,0 +1,4 @@
+<a class="btn btn-primary" href="{% url 'captura:create_egreso' %}" id="create_estudio">
+    <i class="glyphicon glyphicon-plus"></i>
+    Agregar egreso
+</a>
```

Comment:
They can be merged back, this separate structure was implemented when creating wasn't separated from editing, I will return this to the dashboard_transaccion